scripts/api_test.sh

```diff
@@ -670,7 +670,7 @@ function run_tests() {
 	API_netPeerCount_test
 	API_getBalance_test
 	API_getStorageAt_test
-	API_getTransactionCount_test
+	API_getAccountNonce_test
 	API_sendRawTransaction_test
```

Comment:
Is it possible to add api tests for transaction and staking transaction counts as well?

Reply:
sure